modules/ROOT/pages/clustering/clustering-advanced/unbind.adoc

```diff
@@ -12,7 +12,9 @@ Therefore, it is strongly recommended to consult with Neo4j Support before using
 
 Use the `unbind` command only when troubleshooting **a specific server** and remember there is no guarantee that the allocator will reassign the same databases to this server, potentially resulting in orphaned database stores.
 
-The `unbind` command preserves all database stores on the server; and when the unbound server is restarted and enabled, it is seen as an entirely new server.
+The `unbind` command preserves all database stores on the server; and when the unbound server is restarted, it is seen as an entirely new server - so it won't host any of the database it did host before the operation.
+
+The `unbind` command can't be used anymore to convert a cluster member into a standalone server. For that we recommend taking backups, create the standalone server and the use those backups to create the databases.
```

Comment:
```suggestion
In 2025.x, the `unbind` command cannot be used to convert a cluster member into a standalone server.
Instead, it is recommended to take backups, create the standalone server, and then use those backups to restore the databases.
```

Reply:
For the 5.x docs, I'll use 'In 5.x series, the `unbind` command cannot be used...'